include/RaylibException.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef RAYLIB_CPP_INCLUDE_RAYLIBEXCEPTION_HPP_
+#define RAYLIB_CPP_INCLUDE_RAYLIBEXCEPTION_HPP_
+
+#include <stdexcept>
+
+#include <string>
+
+#include "raylib.h"
```

Comment:
```suggestion
#include "./raylib.hpp"
```
Is there a reason that you are using `#include "raylib.h"` here instead of your `#include "./raylib.hpp"` you are using elsewhere? It just has the small implication in my project that I have to change the path for `raylib.h` in `raylib.hpp`, `raymath.hpp` and here.

Reply:
Thanks!